Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6.6-stretch
```

Comment:
This file needs a copyright header, same as the other files. Other than that LGTM.